src/utilities/cities/index.ts

```diff
@@ -0,0 +1,24 @@
+import { CITIES_DATA } from '../../common/cities';
+
+import { State, getStates } from '../states';
+
+const sortAlphabetically = (cityA: string, cityB: string) => cityA.localeCompare(cityB);
+
+export function getCities({ state: stateFilter }: { state?: Partial<State> } = {}): string[] {
```

Comment:
Maybe we should infer if we search by name or code based on received type: if string we search by name, if we receive a number we search by code.

I this way that you propose, we can receive `code` and `name` together and it would be a little inconsistent.